qiskit/algorithms/eigensolvers/ssvqe.py

```diff
@@ -0,0 +1,575 @@
+                If ``None``, then SSVQE will set these to be a list of mutually orthogonal
+                computational basis states.
+            weight_vector: An optional list or array of real positive numbers with length
+                equal to the value of *num_states* to be used in the weighted energy summation
+                objective function. This fixes the ordering of the returned eigenstate/eigenvalue
+                pairs. If ``None``, then SSVQE will default to [n, n-1, ..., 1] for `k` = n.
+            gradient: An optional gradient function or operator for optimizer.
+            callback: A callback that can access the intermediate data at each optimization step.
+                These data are: the evaluation count, the optimizer ansatz parameters,
+                the evaluated mean energies, and the metadata dictionary.
+            check_input_states_orthogonality: A boolean that sets whether or not to check
+                that the value of initial_states passed consists of a mutually orthogonal
+                set of states. If ``True``, then SSVQE will check that these states are mutually
+                orthogonal and return an error if they are not. This is set to ``True`` by default,
+                but setting this to ``False`` may be desirable for larger numbers of qubits to avoid
+                exponentially large computational overhead before the simulation even starts.
+        """
+
+        super().__init__()
+
+        self.k = k
+        self.initial_states = initial_states
+        self.weight_vector = weight_vector
+        self.ansatz = ansatz
+        self.optimizer = optimizer
+        self.initial_point = initial_point
+        self.gradient = gradient
+        self.callback = callback
+        self.estimator = estimator
+        self.check_initial_states_orthogonal = check_input_states_orthogonality
+
+    @property
+    def initial_point(self) -> Sequence[float] | None:
+        """Returns initial point"""
+        return self._initial_point
+
+    @initial_point.setter
+    def initial_point(self, initial_point: Sequence[float] | None):
+        """Sets initial point"""
+        self._initial_point = initial_point
+
+    @classmethod
+    def supports_aux_operators(cls) -> bool:
+        return True
+
+    def compute_eigenvalues(
+        self,
+        operator: BaseOperator | PauliSumOp,
+        aux_operators: ListOrDict[BaseOperator | PauliSumOp] | None = None,
+    ) -> EigensolverResult:
+
+        ansatz = self._check_operator_ansatz(operator)
+
+        initial_point = _validate_initial_point(self.initial_point, ansatz)
+
+        initial_states = self._check_operator_initial_states(self.initial_states, operator)
+
+        bounds = _validate_bounds(ansatz)
+
+        initialized_ansatz_list = [initial_states[n].compose(ansatz) for n in range(self.k)]
+
+        self.weight_vector = self._check_weight_vector(self.weight_vector)
+
+        start_time = time()
```

Comment:
This line seems to be overwritten by line 238.

Reply:
Okay, I have now fixed this.